README.md

```diff
@@ -191,22 +179,42 @@ point to the directory named `sysroots` in the poky toolchain. You will
 likely want `COMPUTECPP_BITCODE=spir32`. Otherwise, these instructions
 should still work.
 
-## Troubleshooting
+## Contributions
 
-The master branch of SYCL-DNN should always compile and tests should always
-pass on our supported platforms. Ideally we should be writing portable,
-standards-compliant SYCL code, and as such it should pass tests on all
-compatible OpenCL hardware. See CONTRIBUTING.md for details about creating
-bug reports for this project.
+Please see the file [CONTRIBUTING.md]((CONTRIBUTING.md)) for further details if 
```

Comment:
Thanks for adding the inks to the contributing file. It looks like this the extra parentheses cause an invalid link to be generated.

```suggestion
Please see the file [CONTRIBUTING.md](CONTRIBUTING.md) for further details if 
```